### PR TITLE
Issue 125 gradient

### PR DESCRIPTION
**Pull request by Jegp**
Fixes issues with gradients not flowing correctly #125

### CODE REVIEW

File: norse/torch/module/lif_mc.py [hunk 1]
```diff
@@ -93,6 +93,7 @@ def forward(
                     dtype=input_tensor.dtype,
                 ),
             )
+            state.v.requires_grade = True
```
Comment (cpehle) [inline norse/torch/module/lif_mc.py:96]: Spelling error?

**Review by cpehle — CHANGES_REQUESTED** [review]
LGTM (apart from typo). We should also adopt the convention used in LIFFeedForwardCell for the other Forward cells.

File: norse/torch/module/lif_mc.py (hunk 1 above)
Comment (Jegp) [reply norse/torch/module/lif_mc.py]: Indeed. And unnecessary so I deleted it

**Review by cpehle — APPROVED** [review]
